introduction/what-is.md

```diff
@@ -0,0 +1,45 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+The basic idea of InnerSource is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
+The producing team partners to review and accept the pull request and the functionality it implements.
+In this context of a code submission, we call the consuming team the **Guest** team and the producing team the **Host** team.
```

Comment:
Hmm ... I think the terminology is right, but some readers might get confused here, because the **Guest** team actually created the PR and thus _produced_ code, rather than the **Host** team, who, in the best case, just _consumed_ the PR. Maybe there's a way to frame this differently?!

Reply:
Updated to _consuming_  and _producing_ to _submitting_ and receiving_.